tests/resources/test_dataset.py

```diff
@@ -349,6 +350,11 @@ def test_gemd_posts(dataset):
         dataset.delete(updated)
         assert dataset.session.calls[-1].path.split("/")[-3] == basename(collection._path_template)
 
+        # But they would have dispatched differently w/ just a uid
+        dataset.session.set_response(updated.dump())
+        dataset.delete(updated.uid)
+        assert dataset.session.calls[-1].path.split("/")[-3] == basename(GEMDResourceCollection._path_template)
+
```

Comment:
Test uid v. object branching.  Not actually registered, so repeat delete isn't a problem.